activity_browser/app/ui/web/startscreen/welcome.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html><head>
+<meta content="text/html; charset=ISO-8859-1" http-equiv="content-type"><title>Welcome</title>
+<style type="text/css"></style>
+<style>
+body {background-color: white; font-family:Calibri;}
```

Comment:
I think it would nice to use the font of the Qt Application (which i think is somehow determined by the OS, so this could be tricky). With Calibri I feel like the text looks a bit out of place compared to the rest of the window.